vibes/vibed-ui/src/platform/linux/LinuxPlatform.cpp

```
@@ -0,0 +1,165 @@
+                break;
+            case MotionNotify:
+                queuedEvent.type = core::EventType::MouseMove;
+                queuedEvent.x = event.xmotion.x;
+                queuedEvent.y = event.xmotion.y;
+                shouldQueue = true;
+                break;
+            case KeyPress:
+                queuedEvent.type = core::EventType::KeyDown;
+                queuedEvent.textCode = 0;
+                {
+                    char textBuffer[8] = {0};
```

Clang (14.0.6)

[warning] do not declare C-style arrays, use std::array<> instead

(modernize-avoid-c-arrays)